content/components/filter-input.mdx

```diff
@@ -0,0 +1,318 @@
+    />
+    <Caption>Don't use the `state` variant when filtering pull requests.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Highlighting
+
+If invalid values are used we don't show highlighting.
+
+While this applies to all qualifiers we do understand that for the ones that depend on our api endpoints (`users`, `repositories`, `organisations`...) we can't always accurately match every result and therefore while not ideal we can always highlight those. For example when not single signed-on to a organisation some repositories might not be returned.
+
+Also consider that users might copy paste strings containing qualifiers. In this case you need to run through every qualifier to set the highlight if needed.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/980622/205069299-569989df-e6b7-410f-bcf6-5106774887a0.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Remove highlighting on values when they are invalid.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/980622/205069297-b9e03b40-0d52-4079-966d-e6d089440ab6.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Don't show highlighting on invalid values.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### No matches
+
+When no matches are found we stop suggesting items through the popover and don't show that there aren't any matches.
+In rare cases the user could be searching for `format:format` within the content of items they are filtering to and this will still allow them to do so.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/980622/205070358-9e160253-d3d5-4354-809b-ff52071eea1e.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Stop suggesting if no matches are found.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/980622/205070353-845bf3b5-de35-4c17-86f5-9d4f953b935a.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Don't show a empty state when no matches are found.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Values with spaces
+
+Some qualifiers like `location:` might have values which contain spaces. eg: `Amsterdam, Netherlands`. In this we encapsulate the value between straight quotes (not smart quotes).
+
+Without this it would be hard for the parser to understand where the value ends.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/980622/205074856-ae4b1b8b-7276-4d6d-8177-baa465241b30.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Use straight quotes around values with spaces.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/980622/205074852-1b3376d9-aee1-4ba3-994b-84305604e0bf.png"
+      role="presentation"
+      width="456"
+    />
+    <Caption>Don't use quotes around values with spaces.</Caption>
```

Comment:
I don't get it. Should I use quotes or not?

Reply:
Haha whoops that's some true inception here.

Replaced with `Don't use values with spaces without quotes`